tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/BallerinaProgramHelper.java

```diff
@@ -78,6 +78,26 @@ public static String createVariableWithEmptyMap(BallerinaASTModelBuilder balleri
     }
 
     /**
+     * Create ballerina map and initialize it with values
+     *
+     * @param ballerinaASTModelBuilder
+     * @param parameters
+     */
+    public static void createAndInitializeMap(BallerinaASTModelBuilder ballerinaASTModelBuilder,
+            Map<String, Object> parameters) {
+        ballerinaASTModelBuilder.addTypes(Constant.BLANG_TYPE_MAP);
+        ballerinaASTModelBuilder.startMapStructLiteral();
+        for (Map.Entry<String, Object> entry : parameters.entrySet()) {
+            ballerinaASTModelBuilder.createStringLiteral(entry.getKey());
+            ballerinaASTModelBuilder.createStringLiteral((String) entry.getValue());
+        }
+        for (int i = 0; i < parameters.size(); i++) {
+            ballerinaASTModelBuilder.addMapStructKeyValue();
```

Comment:
Any reason why we can't add the map struct key value in the above loop in ln 90 ?